views/layouts/default.hbs

```diff
@@ -5,11 +5,8 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>{{ title }}</title>
     <link href="https://fonts.googleapis.com/css?family=Fira+Sans:300,400,500,600" rel="stylesheet" />
-
     <link rel="stylesheet" href="/css/vendor/foundation.min.css">
     <link rel="stylesheet" href="/css/app.css">
-    <script type="text/javascript" src="/js/vendor/jquery.js" defer></script>
-    <script type="text/javascript" src="/js/vendor/foundation.min.js" defer></script>
```

Comment:
Are we getting rid of these for good, or just temporarily?
If the former, should we nuke the [/public/js/vendor](https://github.com/mozilla/blurts-server/tree/master/public/js/vendor) directory?

Reply:
They can go as far I'm concerned. I have been nervous about pruning ancestral code/imports/templates but will go through and clean house today. Might be able to get rid of the foundation css too!